### PR TITLE
Add a copy with method to FeedbackThemeData

### PR DESCRIPTION
**Pull request by ricardoboss**
## :scroll: Description
Adds a `copyWith` method to `FeedbackThemeData`. Also adds `brightness` as a settable property.

## :bulb: Motivation and Context
This fixes issue #311.

## :green_heart: How did you test it?
By calling `copyWith` in the example and seeing its results.

## :pencil: Checklist

- [x] I reviewed submitted code
- [x] I added tests to verify changes
- [ ] I updated the docs if needed
- [x] All tests passing
- [x] No breaking changes

### CODE REVIEW

File: feedback/lib/src/theme/feedback_theme.dart [hunk 1]
```diff
@@ -122,6 +126,39 @@ class FeedbackThemeData {
 
   /// [ColorScheme] on the feedback UI
   late final ColorScheme colorScheme;
+
+  /// Creates a copy of the current [FeedbackThemeData] with the given
+  /// optional fields replaced with the given values.
+  FeedbackThemeData copyWith({
```
Comment (ueman) [inline feedback/lib/src/theme/feedback_theme.dart:132]: Can you add a test for this?

**Review by ueman — COMMENTED** [review]
The changes look good to me, thanks!

File: feedback/lib/src/theme/feedback_theme.dart (hunk 1 above)
Comment (ricardoboss) [reply feedback/lib/src/theme/feedback_theme.dart:132]: @ueman done

Comment (ueman) [reply feedback/lib/src/theme/feedback_theme.dart:132]: Thank you!

**Pull request merged**